documentation.html

```diff
@@ -392,8 +409,55 @@ <h4 class="display-inlineBlock">Toggle Button as Radio Button</h4>
               </div>
             </div>
           </section>
-          
+
         </section>
+
+
+
+        <!-- Dark button class starts here -->
+
+        <section class="instruction-block">
```

Comment:
Add id here with URL same as href in the side menu

Reply:
i actually lost all the old files i had because i switched from windows to linux, but i managed to add these changes to my old forked repository ,now i cant commit it in git, im getting a fatal error when i do this, is there any way to do this?will keep trying